app/models/osco.rb

```diff
@@ -0,0 +1,5 @@
+module Osco
+  def self.table_name_prefix
+    'osco_'
```

Comment:
adding prefix for the table and module for the class. I thought I did this already?